package.json

```diff
@@ -12,7 +12,9 @@
     "build-tests": "babel --out-dir build --quiet test/index.spec.js",
     "lint": "eslint .",
     "prepare": "npm run build",
-    "test": "jest && npm run test:documentation",
+    "test": "npm run test-jest && npm run test-mocha && npm run test:documentation",
+    "test-jest": "jest",
+    "test-mocha": "mocha -r mocha-dominate -r babel-register --opts ./test/mocha.empty.opts",
```

Comment:
Why is it necessary to point to empty opts?

Reply:
I can make it work by adding the config to the existing opts file.

Reply:
<img width="1124" alt="Screenshot 2019-06-26 at 20 42 21" src="https://user-images.githubusercontent.com/90802/60206017-f237e380-9852-11e9-8aa3-c2eee38f8ac9.png">

Reply:
It is still super fast.